### PR TITLE
Add Docker image buidling

### PR DESCRIPTION
**Pull request by atd**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -35,6 +36,10 @@ group :development do
   gem 'spring-watcher-listen', '~> 2.0.0'
 end
 
+group :production do
+    gem 'pg'
```
Comment (nogates) [inline Gemfile]: indentation!! :trollface:

File: config/routes.rb [hunk 2]
```diff
@@ -5,5 +5,7 @@
   resources :turns
   resources :groups
 
-  devise_for :users
+  unless ENV['PRECOMPILE']
```
Comment (nogates) [inline config/routes.rb:8]: what is this?  is it failing when compiling the assets?

**Review by nogates — APPROVED** [review]
awesome! so does this get send to dockerhub every time we merge a PR?

File: Gemfile (hunk 1 above)
Comment (atd) [reply Gemfile]: Upssss

File: config/routes.rb (hunk 2 above)
Comment (atd) [reply config/routes.rb:8]: Devise requires ActiveRecord, which tries to load the database configuration in production

**Comment by atd** [comment]
It seems so: https://hub.docker.com/r/singularitiesorg/circular-works/

What do you think about it? Probably too many builds?

File: config/routes.rb (hunk 2 above)
Comment (nogates) [reply config/routes.rb:8]: :+1:

**Comment by nogates** [comment]
> What do you think about it? Probably too many builds?

well, sometimes it's useful to have a docker image build when working on a branch / PR... if it doesn't become a problem we can leave it as it's now

**Comment by atd** [comment]
:+1: 

We have to discuss about the deployment; currently Travis passes the tests faster than Docker hub builds the image, so the travis' hook arrives our samson before the docker image is ready

**Pull request merged**